recipes/pyexplainer/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "pyexplainer" %}
+{% set version = "0.1.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c5ea63282519b87b5d3bafe41e6b6850e7f594631ecae5ae894c687afa0be470
+
+build:
+  script: "{{ PYTHON }} -m pip install . -vv"
+  number: 0
+  skip: true  # [py<38]
+  
+requirements:
+  host:
+    - python
+    - pip
+    - poetry
+  run:
+    - python
+    - scikit-learn
+    - numpy
+    - scipy
+    - ipywidgets
+    - ipython
+    - pandas
+
+test:
+  imports:
+    - pyexplainer   
+
+about:
+  home: https://github.com/awsm-research/pyExplainer
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGPL.
+  # Use the SPDX identifier, e.g: GPL-2.0-only instead of GNU General Public License version 2.0
+  # See https://spdx.org/licenses/
+  license: MIT
+  # The license_family, i.e. "BSD" if license is "BSD-3-Clause". (optional)
+  license_family: MIT
+  # It is strongly encouraged to include a license file in the package,
+  # (even if the license doesn't require it) using the license_file entry.
+  # See https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#license-file
```

Comment:
Can you please remove all the comments down here?

Reply:
sure